File: SharedApi/Models/ZolikType.cs
using System;
using System.Collections.Generic;
using System.Text;
using SharedLibrary.Interfaces;

namespace SharedApi.Models
{
	public partial class ZolikType : IZolikType
	{
		public int ID { get; set; }
		public int? SplitsToID { get; set; }
		public string Name { get; set; }
		public string FriendlyName { get; set; }
		public bool IsSplittable { get; set; }
		public bool IsTestType { get; set; }
		public bool AllowGive { get; set; }
	}
}

File: MobileApp/zoliky_teachers/lib/utils/api/models/ZolikType.dart
import 'package:zoliky_teachers/utils/interfaces/IDbObject.dart';

class ZolikType implements IDbObject {
  int id;
  //String name;
  String friendlyName;
  bool isSplittable;
  bool isTestType;
  bool allowGive;

  ZolikType();

  ZolikType.fromJson(Map<String, dynamic> json)
      : id = json["ID"],
    //    name = json["Name"],
        friendlyName = json["FriendlyName"],
        isSplittable = json["IsSplittable"],
        isTestType = json["IsTestType"],
        allowGive = json["AllowGive"];
}
